test/SILOptimizer/loweraggregateinstrs_expandall.sil

```diff
@@ -37,7 +37,7 @@ struct S {
 }
 
 enum E {
-  case NoElement()
+  case NoElement(Void)
```

Comment:
Nitpick: this one should probably go to `case NoElement` instead.

Reply:
Thanks @jrose-apple, but I don't quite understand.  This change is needed or else the test fails.  The compiler now complains about the missing `Void`.

Reply:
It's unclear whether the test is supposed to be testing a case with *no* payload or a case with a *zero-sized* payload, but given the name of the enum case I'm guessing it's the former.

Reply:
I think it's the latter. @CodaFi banned enums with an empty tuple payload so that we can stage in the 'normalize enum cases' proposal later without breaking source compatibility, and I recall discussing the intent behind this test case at the time.